static_frame/core/node_iter.py

```diff
@@ -118,15 +127,15 @@ def _apply_iter_items_parallel(self,
 
         # use side effect list population to create keys when iterating over values
         func_keys = []
-        arg_gen: PoolArgGen
+        # arg_gen: PoolArgGen
```

Comment:
FYI this is the only reference to `PoolArgGen`, so if this change is permanent that type can be removed

Reply:
Good catch!